From a13d1b312d4eeeac60d6a03ed9ffd54fa876eccc Mon Sep 17 00:00:00 2001
From: Aurelian Zanoschi <Aurelian.Zanoschi@freescale.com>
Date: Fri, 3 May 2013 09:36:43 +0300
Subject: [PATCH 155/430] dpa_stats: added missing header dependency for
 qman_fq structure

The DPA Stats component uses the qman_fq structure from QMan. This
structure uses rb_node defined in rbtree.h, but doesn't include it, so
it was needed to include rbtree.h or module.h into dpa_stats.

The DPA Stats component uses the qman_fq structure from QMan. This
structure uses rb_node defined in rbtree.h, but doesn't include it, so
it was needed to include rbtree.h or module.h into dpa_stats.

Change-Id: I79caee058a7a0a4849cfd1827ff7fdb50d4d8d75
Signed-off-by: Aurelian Zanoschi <Aurelian.Zanoschi@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/2338
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/staging/fsl_dpa_offload/dpa_stats.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_stats.c b/drivers/staging/fsl_dpa_offload/dpa_stats.c
index 8a524f1..c54c30d 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_stats.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_stats.c
@@ -33,6 +33,7 @@
  * DPA Statistics Application Programming Interface implementation
  */
 
+#include <linux/module.h>
 #include <linux/of_platform.h>
 #include "lnxwrp_fm.h"
 #include "dpaa_eth.h"
-- 
1.7.5.4

